src/ApplicationLogs/LogReader.cs

```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2016-2021 The Neo Project.
```

Comment:
https://github.com/neo-project/neo/blob/6b0b7e8efcb3bb390168472482759e4c70ac7ca9/src/neo/Cryptography/Base58.cs#L1

How about keeping it consistent with neo?